docs/UserGuide.md

```diff
@@ -42,124 +42,125 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
-* `view /member 10` displays members of the 10th member in the member list.
+* `viewMember 1` displays members of the 1st member in the member list.
+* `viewMember 10` displays members of the 10th member in the member list.
 
-### Create an Event : `create /event`
+### Create an Event : `createEvent`
 
 Creates a new event and adds it to the database.
 
-Format: `create /event n/EVENT_NAME [l/LOCATION] [d/DATE]`
+Format: `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE]`
 
 Examples:
-* `create /event n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
-* `create /event n/Combined Hall Ensemble Concert d/16-02-2024` creates an event `Combined Hall Ensemble Concert` in CCACommander.
+* `createEvent n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
+* `createEvent n/Combined Hall Ensemble Concert d/16-02-2024` creates an event `Combined Hall Ensemble Concert` in CCACommander.
 
-### Delete an Event: `delete /event`
+### Delete an Event: `deleteEvent`
 
 Deletes the event at the specified index.
 
-Format: `delete /event EVENT_INDEX`
+Format: `deleteEvent EVENT_INDEX`
 
 * Deletes the event at the specified `EVENT_INDEX`.
 * The index refers to the index number shown in the displayed event list.
 * The index **must be a positive integer** that is within the range of the length of the event list.
 
 Examples:
-* `delete /event 1` deletes the 1st event in the event list.
-* `delete /event 10` deletes the 10th event in the event list.
+* `deleteEvent 1` deletes the 1st event in the event list.
+* `deleteEvent 10` deletes the 10th event in the event list.
 
-### Add Member to an Event: `add /member /event`
+### Add Member to an Event: `enrol`
 
 Adds a member to an event.
 
-Format: `add /member MEMBER_INDEX /event EVENT_INDEX`
+Format: `enrol m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]`
 
-* Adds the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX`.
+* Adds the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARKS` stating extra remarks about the member and event.
 * The `MEMBER_INDEX`/`EVENT_INDEX` refers to the index number shown in the displayed member/event list.
 * The `MEMBER_INDEX`/`EVENT_INDEX` **must be a positive integer** that is within the range of the length of the member/event list.
+* The `NUMBER_OF_HOURS` **must be a positive integer**.
 
 Examples:
-* `add /member 1 /event 5` adds the 1st member in the member list to the 5th event in the event list.
-* `add /member 5 /event 1` adds the 5th member in the member list to the 1st event in the event list.
+* `enrol m/1 e/5 h/3 r/did planning` adds the 1st member in the member list to the 5th event in the event list, where the member had 3 hours of contributions to that event and has a remark stating that he/she "did planning"".
```

Comment:
Would 'they' fit better to include members of all genders?
```suggestion
* `enrol m/1 e/5 h/3 r/did planning` adds the 1st member in the member list to the 5th event in the event list, where the member had 3 hours of contributions to that event and has a remark stating that they "did planning"".
```

Reply:
I think that changing it just "member" will be the least troublesome way to resolve this.